integration-tests-ex/cases/src/test/java/org/apache/druid/testsEx/indexer/ITAzureV2ParallelIndexTest.java

```diff
@@ -49,4 +49,13 @@ public void testAzureIndexData(Pair<String, List<?>> azureInputSource) throws Ex
   {
     doTest(azureInputSource, new Pair<>(false, false), "azureStorage");
   }
+
+
+  @Test
+  @Parameters(method = "resources")
+  public void testAzureIndexData_kill(Pair<String, List<?>> azureInputSource) throws Exception
+  {
+    String dataSource = doTest(azureInputSource, new Pair<>(false, false), "azureStorage");
+    AbstractAzureInputSourceParallelIndexTest.validateAzureSegmentFilesDeleted("segments" + "/" + dataSource);
```

Comment:
Why not validate the segment files are deleted in the test above so we don't need to run this twice.

Reply:
Also, how are the segments being deleted? I tried tracing the `doTest` and I don't see it issuing a kill task to delete the segments from deep storage

Reply:
i wasn't sure if that was overloading the definition of the test but i can do that

Reply:
Since the integration tests take a long time to run, and this test is parameterized, I think it is good to try to minimize the run time by overloading the test in this case.